revolut/__init__.py

```diff
@@ -244,6 +294,8 @@ def send(self, dest, amount, currency, request_id, reference=None):
                     )
                 )
             receiver = {"counterparty_id": cpt.id}
+        if not receiver:
+            raise ValueError("Receiver cannot be empty")
```

Comment:
To w ogóle zachodzi?

Poza tym użytkownik nie podaje parametru `receiver` więc taki komunikat tylko go zdezorientuje.